examples/api/test/material/flexible_space_bar/flexible_space_bar.0_test.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/flexible_space_bar/flexible_space_bar.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  // The app being tested loads images via HTTP which the test
+  // framework defeats by default.
+  setUpAll(() {
+    HttpOverrides.global = null;
+  });
+
+  testWidgets('The app bar should stretch when over-scrolled', (WidgetTester tester) async {
```

Comment:
```suggestion
  testWidgets('The app bar stretches when over-scrolled', (WidgetTester tester) async {
```

I was used to add 'should' on test descriptions but it looks better without it. :smile: